src/Cpp/tests/src/SpatialStructures.cpp

```diff
@@ -2028,7 +2047,7 @@ namespace CInterfaceTests {
 
 		// By the postconditions of GetNodeAttributes, this should update scores_out,
 		// and scores_out_size with the variables we need
-		GetNodeAttributes(&g, attr_type.c_str(), scores_out, &scores_out_size);
+		GetNodeAttributes(&g, NULL, attr_type.c_str(), 0, scores_out, &scores_out_size);
```

Comment:
Split this test into a new one, so the old one stays the same, but a new one allows this additional argument.